pom.xml

```diff
@@ -491,7 +491,9 @@
                   <version>[3.0.4,)</version>
                   <message>Maven 3.0 through 3.0.3 inclusive do not pass correct settings.xml to Maven Release Plugin.</message>
                 </requireMavenVersion>
-                <requirePluginVersions />
+                <requirePluginVersions>
+                  <banSnapshots>false</banSnapshots>
```

Comment:
Otherwise we cannot even test development versions of plugins. Not necessary to enforce this since `release:prepare` would do so anyway.